### DISCUSSION TITLE
How do I customize the terminal colors?

### DISCUSSION THREAD

**Discussion by cdarken** [neanias/everforest-nvim#30] [Q&A] [answered]
Hi. I'm using p10k for my  zsh prompt and the text is very low contrast in the nvim terminal while using everforest. This doesn't happen when using tokyonight for instance.
Is there a way to override some colors to increase the contrast. Maybe using `on_highlights` callback? Which values should I replace if so?
Here's a screenshot with my terminal in everforest for exemplification:
![image](https://github.com/user-attachments/assets/2548380d-3dfa-4ba2-bc65-cc20d559db47)
and one with tokyonight:
![image](https://github.com/user-attachments/assets/f5970abf-3827-4bad-a69a-805be72b1c27)

**Comment by neanias** [comment]
Hi there!

I've just updated the terminal colours to have better contrast (both light & dark were the same). Could you update the plugin and see if that's better?

**Comment by neanias** [comment]
Also, for customising the terminal colours, you can set the `vim.g.terminal_color_0` to `vim.g.terminal_color_15` variables.

**Comment by cdarken** [comment, marked as answer]
Thanks for answering! The new version didn't change anything visible, but setting `vim.g.terminal_color_0` to a darker shade fixed the contrast issue in the git branch display.

**Reply by neanias** [reply]
Ah, that was the one I didn't update 🤦 Glad you worked it out though!
